lib/modules/whisper/js/embarkjs.js

```diff
@@ -13,29 +13,24 @@ __embarkWhisperNewWeb3.setProvider = function (options) {
   }
   // TODO: take into account type
   self.web3 = new Web3(new Web3.providers.WebsocketProvider("ws://" + provider, options.providerOptions));
-  self.web3.currentProvider.on('connect', () => {
```

Comment:
Why get rid of the on connect?

Reply:
Not intentional. I think I lost this commit (https://github.com/embark-framework/embark/commit/fe3e4690f6dfaf361aa0e671ed7003b09c024911) during a rebase.